exercises/practice/armstrong-numbers/tests/Template.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { Blockchain, SandboxContract } from '@ton-community/sandbox';
+import { Cell, toNano } from 'ton-core';
+import { Template } from '../wrappers/Template';
+import '@ton-community/test-utils';
+import { compile } from '@ton-community/blueprint';
+
+describe('Template', () => {
+    let code: Cell;
+
+    beforeAll(async () => {
+        code = await compile('Template');
+    });
+
+    let blockchain: Blockchain;
+    let template: SandboxContract<Template>;
+
+    beforeEach(async () => {
+        blockchain = await Blockchain.create();
+
+        template = blockchain.openContract(Template.createFromConfig({}, code));
+
+        const deployer = await blockchain.treasury('deployer');
+
+        const deployResult = await template.sendDeploy(deployer.getSender(), toNano('0.05'));
+
+        expect(deployResult.transactions).toHaveTransaction({
+            from: deployer.address,
+            to: template.address,
+            deploy: true,
+            success: true,
+        });
+    });
+
+    let True: number = -1; // True definition in FunC
+    let False: number = 0; // False definition in FunC
+
+    it('zero is an armstrong number', async () => {
+        expect(await template.get_is_armstrong_number(0)).toBe(True);
+    });
+
+    it('single digit numbers are armstrong numbers', async () => {
```

Comment:
On Exercism, we try to have the student work on exercises using Test-Driven Development. One way in which we encourage that is to, by default, only enable the first test. The idea being that once the student has solved that, they can then move on to the next test. In practice this would mean that all but the first test would be defined using `xit` (which will skip that test):

```suggestion
    xit('single digit numbers are armstrong numbers', async () => {
```